rpcserver.go

```diff
@@ -2570,80 +2570,92 @@ func (r *rpcServer) CommitVirtualPsbts(ctx context.Context,
 		return nil, fmt.Errorf("error serializing packet: %w", err)
 	}
 
-	// The change output and fee parameters of this RPC are identical to the
-	// walletrpc.FundPsbt, so we just map them 1:1 and let lnd do the
-	// validation.
-	coinSelect := &walletrpc.PsbtCoinSelect{
-		Psbt: packetBytes,
-	}
-	fundRequest := &walletrpc.FundPsbtRequest{
-		Template: &walletrpc.FundPsbtRequest_CoinSelect{
-			CoinSelect: coinSelect,
-		},
-		MinConfs:              1,
-		ChangeType:            P2TRChangeType,
-		CustomLockId:          req.CustomLockId,
-		LockExpirationSeconds: req.LockExpirationSeconds,
-	}
+	var (
+		lockedUTXO      []*walletrpc.UtxoLease
+		lockedOutpoints []wire.OutPoint
+	)
 
-	// Unfortunately we can't use the same RPC types, so we have to do a
-	// 1:1 mapping to the walletrpc types for the anchor change output and
-	// fee "oneof" fields.
-	type existingIndex = walletrpc.PsbtCoinSelect_ExistingOutputIndex
-	switch change := req.AnchorChangeOutput.(type) {
-	case *wrpc.CommitVirtualPsbtsRequest_ExistingOutputIndex:
-		coinSelect.ChangeOutput = &existingIndex{
-			ExistingOutputIndex: change.ExistingOutputIndex,
-		}
+	fundedPacket := pkt
+	changeIndex := int32(-1)
+	success := false
 
-	case *wrpc.CommitVirtualPsbtsRequest_Add:
-		coinSelect.ChangeOutput = &walletrpc.PsbtCoinSelect_Add{
-			Add: change.Add,
+	if !req.SkipFunding {
+		// The change output and fee parameters of this RPC are
+		// identical to the walletrpc.FundPsbt, so we just map them 1:1
+		// and let lnd do the validation.
+		coinSelect := &walletrpc.PsbtCoinSelect{
+			Psbt: packetBytes,
 		}
+		fundRequest := &walletrpc.FundPsbtRequest{
+			Template: &walletrpc.FundPsbtRequest_CoinSelect{
+				CoinSelect: coinSelect,
+			},
+			MinConfs:              1,
+			ChangeType:            P2TRChangeType,
+			CustomLockId:          req.CustomLockId,
+			LockExpirationSeconds: req.LockExpirationSeconds,
+		}
+
+		// Unfortunately we can't use the same RPC types, so we have to
+		// do a 1:1 mapping to the walletrpc types for the anchor change
+		// output and fee "oneof" fields.
+		type existingIndex = walletrpc.PsbtCoinSelect_ExistingOutputIndex
```

Comment:
We have a `type()` block at the beginning of the file where we create shorter definitions/aliases to avoid these long lines. Could move this and the other offending `walletrpc` types there to avoid the linter complaining.

Reply:
done